src/lib/db/DatabaseManager.ts

```diff
@@ -208,4 +249,118 @@ class DatabaseManager {
   }
 }
 
+function getMoodName(dopamine: number, adrenaline: number, serotonin: number): string {
+  if (dopamine <= 5) {
+    if (adrenaline <= 5) {
+      if (serotonin <= 5) {
+        return "guilt";
+      } else if (serotonin >= 6) {
+        return "content";
+      }
+    } else if (adrenaline >= 6) {
+      if (serotonin <= 5) {
+        return "distress";
+      } else if (serotonin >= 6) {
+        return "relief";
+      }
+    }
+  } else if (dopamine >= 6) {
+    if (adrenaline <= 5) {
+      if (serotonin <= 5) {
+        return "freeze";
+      } else if (serotonin >= 6) {
+        return "joy";
+      }
+    } else if (adrenaline >= 6) {
+      if (serotonin <= 5) {
+        return "fight/flight";
+      } else if (serotonin >= 6) {
+        return "interest";
+      }
+    }
+  }
+  return "content"; // default fallback
+}
```

Comment:
potentially move this out of database manager as it is mood specific utility? in fact it is already in MoodsDisplay.tsx so is it used here? [will not block merge]